docs/README.md

```diff
@@ -0,0 +1,41 @@
+<!--
+ Copyright (c) 2024 Fraunhofer-Gesellschaft zur Förderung der angewandten Forschung e.V.
+
+ This work is licensed under the Fraunhofer License (on the basis of the MIT license)
+ that can be found in the LICENSE file.
+
+ The initial version of this file was auto-generated by Docusaurus. Docusaurus is published under
+ the MIT license.
+ See https://github.com/facebook/docusaurus/blob/6c7f2fde20b2448a1b966dd5f674205c0a08eeba/LICENSE
+ for more information.
+-->
```

Comment:
If we include a license header in this Markdown file an issue should be created to update all markdown files with a license header. (e.g., `webinterface/README.md`)